base/linalg/uniformscaling.jl

```diff
@@ -171,6 +171,15 @@ function isapprox(J1::UniformScaling{T}, J2::UniformScaling{S};
             rtol::Real=Base.rtoldefault(T,S), atol::Real=0, nans::Bool=false) where {T<:Number,S<:Number}
     isapprox(J1.λ, J2.λ, rtol=rtol, atol=atol, nans=nans)
 end
+function isapprox(J::UniformScaling,A::AbstractMatrix;kwargs...)
+    n = checksquare(A)
+    D = diagm(J.λ*ones(typeof(J.λ),n))
+    return isapprox(D,A;kwargs...)
+end
```

Comment:
Or maybe we just don't care about performance of this method too much (since we're just talking about the constant factor here)?  That's plausible.

We could also have a fast (zero-allocating) method for the common case of `norm==vecnorm` and do the slow method everywhere else.